sysutils/coreutils-uutils/Portfile

```diff
@@ -0,0 +1,231 @@
+    wasm-bindgen-backend                               0.2.73  ae70622411ca953215ca6d06d3ebeb1e915f0f6613e3b495122878d7ebec7dae \
+    wasm-bindgen-macro                                 0.2.73  3e734d91443f177bfdb41969de821e15c516931c3c3db3d318fa1b68975d0f6f \
+    wasm-bindgen-macro-support                         0.2.73  d53739ff08c8a68b0fdbcd54c372b8ab800b1449ab3c9d706503bc7dd1621b2c \
+    wasm-bindgen-shared                                0.2.73  d9a543ae66aa233d14bb765ed9af4a33e81b8b58d1584cf1b47ff8cd0b9e4489 \
+    web-sys                                            0.3.50  a905d57e488fec8861446d3393670fb50d27a262344013181c2cdf9fff5481be \
+    wild                                                2.0.4  035793abb854745033f01a07647a79831eba29ec0be377205f2a25b0aa830020 \
+    winapi                                              0.2.8  167dc9d6949a9b857f3451275e911c3f44255842c1f7a76f33c55103a909087a \
+    winapi                                              0.3.9  5c839a674fcd7a98952e593242ea400abe93992746761e38641405d28b00f419 \
+    winapi-build                                        0.1.1  2d315eee3b34aca4797b2da6b13ed88266e6d612562a0c46390af8299fc699bc \
+    winapi-i686-pc-windows-gnu                          0.4.0  ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6 \
+    winapi-util                                         0.1.5  70ec6ce85bb158151cae5e5c87f95a8e97d2c0c4b001223f33a334e3ce5de178 \
+    winapi-x86_64-pc-windows-gnu                        0.4.0  712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f \
+    xattr                                               0.2.2  244c3741f4240ef46274860397c7c74e50eb23624996930e484c16679633a54c
+
+checksums-append    ${distname}${extract.suffix} \
+                    rmd160  f1e67e3bded104a7986b14fb44a49c5b7c85d4c1 \
+                    sha256  9caa4ef91d7604417a7b4c69976293d81234b0d805be1020bdd46d65f63898db \
+                    size    532392
+
+build.pre_args-append --features macos
+
+destroot {
+    xinstall -m 0755 ${worksrcpath}/target/[cargo.rust_platform]/release/coreutils ${destroot}${prefix}/bin/
+    xinstall -m 755 -d ${destroot}${prefix}/libexec/uutils
+
+    foreach binary {
+        arch base32 base64 basename cat chgrp chmod chown chroot cksum comm cp csplit
+        cut date df dircolors dirname du echo env expand expr factor false fmt fold
+        groups hashsum head hostid hostname id install join kill link ln logname ls
+        md5sum mkdir mkfifo mknod mktemp more mv nice nl nohup nproc numfmt od paste
+        pathchk pinky printenv printf ptx pwd readlink realpath relpath rm rmdir seq
+        sha1sum sha224sum sha256sum sha3-224sum sha3-256sum sha3-384sum sha3-512sum
+        sha384sum sha3sum sha512sum shake128sum shake256sum shred shuf sleep sort split
+        stat stdbuf sum sync tac tail tee test timeout touch tr true truncate tsort
+        tty uname unexpand uniq unlink uptime users wc who whoami yes} {
```

Comment:
I think you should avoid having to hardcode a long list of executables here by instead performing a glob on the directory `${destroot}${prefix}/libexec/uutils/ `and linking everything that finds.

Reply:
I have the package building a single, busybox-esque `coreutils` binary. It's not installing anything under libexec, unfortunately. It's possible to build each one individually but then you're back to writing a list of every package into the Portfile. And I think people want a single binary out of this package anyway.

Reply:
If this list is still needed, it might be good to define it as a TCL list outside of the `destroot` phase. And then have your `foreach` loop reference that instead.

Reply:
If you build each utility seperatly then you can during destroy query the install area to see what is available instead of hardcoding the list. For me that is preferable.